### PR TITLE
Fix some tests

### LINKED ISSUE #2676: [Bug] Fix failing tests
Several tests are broken on main

### PR DESCRIPTION
**Pull request by shagunsodhani**
## Motivation

Fixes #2676 

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebookresearch/hydra/blob/master/CONTRIBUTING.md)?

Yes

## Test Plan

CI

## Related Issues and PRs

#2676

### CODE REVIEW

File: tests/test_examples/test_tutorials_basic.py [hunk 1]
```diff
@@ -53,7 +53,8 @@ def test_tutorial_working_directory(tmpdir: Path) -> None:
         "hydra.job.chdir=True",
     ]
     result, _err = run_python_script(cmd)
-    assert result == "Working directory : {}".format(tmpdir)
+    working_directory, output_directory = result.split("\n")
+    assert working_directory == "Working directory : {}".format(tmpdir)
```
Comment (omry) [inline tests/test_examples/test_tutorials_basic.py:57]: Might as well modernize it while you are here.
```suggestion
    assert working_directory == f"Working directory : {tmpdir}"
```

Comment (shagunsodhani) [reply tests/test_examples/test_tutorials_basic.py:57]: You are right! I (mistakingly) thought that f-strings is 3.7+ feature :)

Comment (Jasha10) [reply tests/test_examples/test_tutorials_basic.py]: FYI @shagunsodhani Hydra has basically dropped support for python3.6 at this point.
The `.circleci/config.yml` and `setup.py` files only mention versions 3.7-3.11.

Comment (shagunsodhani) [reply tests/test_examples/test_tutorials_basic.py]: Thanks for the clarification. Should we also remove the references to 3.6 from [nox file](https://github.com/facebookresearch/hydra/blob/main/noxfile.py#L18) and plugins ?

Comment (Jasha10) [reply tests/test_examples/test_tutorials_basic.py]: > Should we also remove the references to 3.6 from [nox file](https://github.com/facebookresearch/hydra/blob/main/noxfile.py#L18) and plugins ?

Sure, I think that would make sense. I'll submit a PR.

**Review by Jasha10 — APPROVED** [review]

**Pull request merged**